### PR TITLE
#14 translate event start date in the local language

### LINKED ISSUE #14: translate start date in calendar blue bar
https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/113

### PR DESCRIPTION
**Pull request by asmaNode**
dateformat

### CODE REVIEW

File: src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js [hunk 1]
```diff
@@ -217,4 +219,10 @@ function calInboxInvitationMessageBlueBarController(
     this.message = message;
     this.meeting = self.meeting;
   }
+
+  function translateDate() {
+    let localMoment = esnI18nService.getLocale();
+    self.event.start.locale(localMoment);
```
Comment (itswillta) [inline src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js]:
```suggestion
    const localMoment = esnI18nService.getLocale();

    self.event.start.locale(localMoment);
```

or

```suggestion
    self.event.start.locale(esnI18nService.getLocale());
```

**Review by itswillta — COMMENTED** [review]
You should add some tests for these changes also.

File: src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js [hunk 2]
```diff
@@ -217,4 +219,10 @@ function calInboxInvitationMessageBlueBarController(
     this.message = message;
     this.meeting = self.meeting;
   }
+
+  function translateDate() {
```
Comment (tuanlc) [inline src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js]:
```suggestion
  function translateStartDate() {
```

**Review by tuanlc — COMMENTED** [review]
Can we write tests for the change?

File: src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js (hunk 2 above)
Comment (asmaNode) [reply src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js]: yes , i will

**Review by rezk2ll — APPROVED** [review]

**Review by MichaelBailly — APPROVED** [review]

File: src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.spec.js [hunk 3]
```diff
@@ -441,4 +450,16 @@ describe('The calInboxInvitationMessageBlueBarController', function() {
       expect(ctrl.isActionable()).to.equal(false);
     });
   });
+
+  describe('The translateStartDate function', function() {
+    it('should translate the format of start date in the local language', function() {
+      var ctrl = initCtrl('REQUEST', '1234', '1');
+      const expectedLocale = 'fr';
```
Comment (tuanlc) [inline src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.spec.js:457]: Mixing `var` & `const` is not cool :wink:

Comment (itswillta) [reply src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.spec.js:457]: You're one day too late :trollface:.

Comment (tuanlc) [reply src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.spec.js:457]: Yep. I know. Just an invest for the future :stuck_out_tongue_winking_eye:

Comment (asmaNode) [reply src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.spec.js:457]: thanks :p

**Pull request merged**